cypress/integration/Change Password Tests/changePassword.specs.js

```diff
@@ -0,0 +1,43 @@
+ /// <reference types="cypress" />
+ import {passwordTab} from  '../../utils/changePasswordTab'
+  describe("NeetoAuth change password functionality", () => {
+    
+  it("Changing current password and trying to valid it can't login anymore with old password",()=>{
+          passwordTab();
+          cy.fixture("credentials").then((userDetails)=>{
+            cy.pwdChange(userDetails.old.password, userDetails.new.password);
+            cy.visit('https://spinkart.neetoauth.net')
+            cy.login(userDetails.correct.email,userDetails.correct.password);
+            cy.loginSubmit(); 
+            cy.get('[data-cy=toastr-message-container]').should('have.text','Something went wrong.');     
+            cy.resetPassword(userDetails.new.password, userDetails.old.password); 
+          }) 
+  })
+
+  it("Changing current password and trying to valid that it can only login with new current password",()=>{
```

Comment:
Please fix the formatting wherever required.
```suggestion
  it("Changing current password and trying to valid that it can only login with new current password", () => {
```